pkg/foxtrot/db.go

```diff
@@ -53,18 +57,84 @@ func (db *db) setupSchema() error {
 
 func (db *db) getUser(ctx context.Context, name string) (*User, error) {
 	u := User{Name: name}
-	selectUserStmt := "SELECT password_hash FROM users WHERE name=?"
-	err := db.conn.QueryRowContext(ctx, selectUserStmt, name).Scan(&u.passwordHash)
+	stmt := "SELECT password_hash FROM users WHERE name=?"
```

Comment:
since you're modifying this line, you may as well add the spaces around the `=` to be consistent with the other statements in this file.

Reply:
yes, sorry, missed that. done.